cli/tauri.js/templates/tauri.js

```diff
@@ -366,28 +363,50 @@ window.tauri = {
     <% } %>
   },
 
-  <% if (ctx.dev) { %>
-  /**
-   * @name setup
-   * @description Inform Rust that the webview has initialized and is
-   * ready for communication
-     */
-  <% } %>
-  setup: function setup() {
-    document.querySelector('body').addEventListener('click', function (e) {
-      var target = e.target;
+  loadAsset: function loadAsset(assetName) {
+    return this.promisified({
+      cmd: 'loadAsset',
+      asset: assetName
+    })
+  }
+};
+
+window.addEventListener('message', function (event) {
```

Comment:
We might want to be able to properly tree shake this out if they don't enable it.

Reply:
oops, this should not be here anymore lol